src/Malt.jl

```diff
@@ -581,18 +661,58 @@ Send an interrupt signal to the worker process. This will interrupt the
 latest request (`remote_call*` or `remote_eval*`) that was sent to the worker.
 """
 function interrupt(w::Worker)
-    if Sys.iswindows()
-        # TODO: not yet implemented
-        @warn "Malt.interrupt is not yet supported on Windows"
-        # _assert_is_running(w)
-        # _send_msg(w, MsgType.from_host_fake_interrupt, (), false)
-        nothing
+    if !isrunning(w)
+        @warn "Tried to interrupt a worker that has already stopped running." summary(w)
     else
-        Base.kill(w.proc, Base.SIGINT)
+        if Sys.iswindows()
+            ccall((:GenerateConsoleCtrlEvent,"Kernel32"), Bool, (UInt32, UInt32), UInt32(1), UInt32(getpid(w.proc)))
```

Comment:
!!!

Reply:
i did a thing 💖

Reply:
and it's working!

Reply:
proof: https://github.com/JuliaPluto/Malt.jl/actions/runs/6324554088/job/17174287331?pr=60#step:5:158